app/models/planning_application.rb

```diff
@@ -41,6 +44,37 @@ class PlanningApplication < ApplicationRecord
     scope status_string.to_sym, -> { where(status: status_string) }
   end
```

Comment:
I think adding AASM means that my metaprogramming hacks to add the getters, setters and scopes are redundant now? What happens if we remove them?

Reply:
Good suggestion - I needed to update some of the methods in the specs to accommodate this, but there is much less duplication now.